.github/ISSUE_TEMPLATE/bug.yaml

```
@@ -0,0 +1,76 @@
+name: 🐞 Bug Report
+description: File a bug report
+title: "[Bug]: <title>"
+labels: [ "bug" ]
+projects: [ "JankariTech/48" ]
+body:
+  - type: markdown
+    attributes:
+      value: Thanks for taking the time to fill out this bug report!
+  - type: checkboxes
+    attributes:
+      label: Is there an existing issue for this?
+      description: Please search to see if an issue already exists for the bug you encountered.
+      options:
+        - label: I have searched for an existing issue
+          required: true
+  - type: textarea
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of what the bug is.\
+  - type: textarea
+    attributes:
+      label: Steps To Reproduce
+      description: Steps to reproduce the behavior.
+      placeholder: |
+        1.
+        2.
+  - type: textarea
+    attributes:
+      label: Expected Behavior
+      description: A concise description of what you expected to happen.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Actual Behavior
+      description: A concise description of what you're experiencing.
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: Examples, Code Samples, or Screenshots
+      description: |
+        Examples, Code Samples, or Screenshots. Anything that will give us more context about the issue you are encountering!
+      Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
+      validations:
+        required: false
+  - type: textarea
+    attributes:
+      label: Environment
+      description: |
+        examples:
+          - **OS**: Ubuntu 22.04
+          - **Node**: 18.17.1
+          - **pnpm**: 8.15.1
+          - **ownCloud Web**: ownCloud Web UI 9.0.0-alpha.1
+          - **oCIS**: Infinite Scale 5.1.0-prealpha+fa57da865 Community
```

if they are the placeholders
```suggestion
          - **ownCloud Web**: ownCloud Web UI 8.0.0
          - **oCIS**: Infinite Scale 5.0.0
```